python/README.md

```diff
@@ -0,0 +1,33 @@
+# do-username
+
+A DigitalOcean-community-themed username generator - the Python version.
+
+https://pypi.org/project/do-username/
+
+## Install
+
+`pip3 install do-username`
+
+## Example
+
+### Using the CLI
+
+`do-username` to generate a username with max length of 30 characters
+
+`do-username 15` to generate a username with max length of 15 characters
+
+### As a python module
+
+```python
+from do_username.generate import generate

```

Comment:
Can we import the `generate` method and the assorted constants into `__init__.py`, so that we can just do `from do_username import generate, SEA_CREATURES` etc.?

Reply:
Yup!